src/components/TodosList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TodoItem from './TodoItem';
+/* eslint-disable */
+class TodosList extends React.Component {
+  render() {
+    return (
+      <ul>
+        {this.props.todos.map((todo) => (
+          <TodoItem
+            key={todo.id}
+            todo={todo}
+            handleChangeProps={this.props.handleChangeProps}
+            deleteTodoProps={this.props.deleteTodoProps}
+            setUpdate={this.props.setUpdate}
+          />
+        ))}
+      </ul>
+    );
+  }
+}
+
+export default TodosList;
```

Comment:
- [ ] I suggest you that remove this `/* eslint-disable */` from your codebase and try to run your project, which is the primary requirement for this project.